azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -142,6 +142,13 @@ def _format_results(self, sampler_seed=None):
 
     @staticmethod
     def _draw_random_sample(sampler_seed, probabilities, shots):
+        _norm = sum(probabilities.values())

```

Comment:
Should we normalize this in other places too? Like when using pass-through or Cirq?

Reply:
Yes, thanks for pointing that out!

Reply:
For cirq, we are using a package `cirq-ionq`. It looks like it does not normalize the probabilities. It would be worth looking into: https://github.com/quantumlib/Cirq/blob/05418b1996129298321435e9a566487968fcf126/cirq-ionq/cirq_ionq/job.py#L230
I suggest following up in that repo though and not in `azure-quantum`. What do you think?